src/commons/application/ApplicationTypes.ts

```diff
@@ -175,7 +175,8 @@ export const defaultDashboard: DashboardState = {
 
 export const defaultAchievement: AchievementState = {
   achievements: [],
-  goals: []
+  goals: [],
+  users: []
```

Comment:
The list of users is not exactly specific to achievements, but.. sigh.

Since it's only used by achievements, fair enough I guess.